docs/reference/developer/adr/0008-consistent-multi-dimensional-coords.rst

```diff
@@ -0,0 +1,49 @@
+ADR 0008: Consistent multi-dimensional coords
+=============================================
+
+- Status: accepted
+- Deciders: Jan-Lukas, Neil, Owen, Simon
+- Date: 2021-08-19
+
+Context
+-------
+
+In the special case of non-dimension coords that have more than 1 dimension, they are considered to be labels for their inner dimension.
+This causes trouble in many cases, e.g., after computation of ``phi`` and ``radius`` for 2-D data from ``x`` and ``y`` coords.
+If for example ``phi.dims = ['x', 'y']`` and ``radius.dims = ['y', 'x']`` then and ``x`` slice will:
```

Comment:
"then and" doesn't make sense in this sentence.

Reply:
fixed